Cargo.toml

```diff
@@ -4,7 +4,7 @@ name = "spiffe"
 # When releasing to crates.io:
 # - Update CHANGELOG.md.
 # - Create a new tag
-version = "0.1.1"
+version = "0.1.2"
```

Comment:
Should it be `0.2.0` since we modified the signature of one method 😬 ?

Reply:
That a question I had myself when I submitted this PR... yeah it probably makes more sense a `0.2.0` version.